### PR TITLE
Elevator default command

### PR DESCRIPTION
**Pull request by LumpyJr**

### CODE REVIEW

File: src/main/java/frc/robot/AngleShooterUtil.java [hunk 1]
```diff
@@ -5,20 +5,20 @@
...
-import frc.robot.subsystems.ShooterSubsystem;
+import frc.robot.subsystems.ElevatorSubsystem;
 
 public class AngleShooterUtil {
     
-    private final ShooterSubsystem s_ShooterSubsystem;
+    private final ElevatorSubsystem s_ElevatorSubsystem;
 
-    private double m_encoderDiff;
+    private static double m_encoderDiff;
```
Comment (notrandomness) [inline src/main/java/frc/robot/AngleShooterUtil.java]: Is there a reason for making this static? If not, we might remove static from this and the updateTargetDiff method.

File: src/main/java/frc/robot/subsystems/ElevatorSubsystem.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    private CANSparkMax elevator;
+
+    public static ElevatorSubsystem getInstance(){
+        if (elevatorSubsystem == null){
+            elevatorSubsystem = new ElevatorSubsystem();
+        } 
+        return (elevatorSubsystem);
+    }    
+
+    public void elevatorSubsystem(){
```
Comment (notrandomness) [inline src/main/java/frc/robot/subsystems/ElevatorSubsystem.java]: Is this supposed to be the constructor? If so, it should be _public ElevatorSubsystem()_

**Review by notrandomness — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/AngleShooterUtil.java (hunk 1 above)
Comment (Crwspy) [reply src/main/java/frc/robot/AngleShooterUtil.java:14]: fixed before

Comment (BucketHatGuy) [reply src/main/java/frc/robot/AngleShooterUtil.java]: fixed

File: src/main/java/frc/robot/subsystems/ElevatorSubsystem.java (hunk 2 above)
Comment (BucketHatGuy) [reply src/main/java/frc/robot/subsystems/ElevatorSubsystem.java]: fixed

**Review by notrandomness — DISMISSED** [review]

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 3]
```diff
@@ -44,15 +44,15 @@ public static ShooterSubsystem getInstance(){
     private ShooterSubsystem(){
         topShooter = new CANSparkMax(20, MotorType.kBrushless);
         bottomShooter = new CANSparkMax(21, MotorType.kBrushless);
+        
         shooterElevator = new CANSparkMax(25, MotorType.kBrushless);
```
Comment (notrandomness) [inline src/main/java/frc/robot/subsystems/ShooterSubsystem.java]: shooterElevator should be removed from the ShooterSubsystem, as well as any elevator related methods. All commands controlling the elevator motor should use the ElevatorSubsystem.

**Review by notrandomness — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/ShooterSubsystem.java (hunk 3 above)
Comment (BucketHatGuy) [reply src/main/java/frc/robot/subsystems/ShooterSubsystem.java]: done

**Review by notrandomness — APPROVED** [review]

**Pull request merged**